book/jsonrpc/trace.md

```diff
@@ -0,0 +1,183 @@
+# `trace` Namespace
+
+<!-- TODO: We should probably document the format of the traces themselves, OE does not do that !-->
```

Comment:
:+1: :+1: agreed - cc @mattsse

Reply:
by format, you mean the return type and all the fields? what they represent?

Reply:
yes, the structure is largely undocumented on the return types.